### DISCUSSION TITLE
Where is serverResolutions for tiling?

### DISCUSSION THREAD

**Discussion by justb4** [openlayers/openlayers#16272] [Q&A] [answered]
In OL 2 a property `serverResolutions` (Array) could be set on tile services for example on `OpenLayers.Layer.WMTS`. This allowed the client to "blow up" tiles where either the server does not support a resolution greater than the maximum in  `serverResolutions` (and `resolutions` Array contained more levels) or simply to reduce network traffic. Especially for pre-rendered raster maps where the highest rendered resolution is that maximum. 

In higher versions of OL like v9 v10, I cannot find `serverResolutions` anymore. Is there an equivalent perhaps?

**Comment by ahocevar** [comment, marked as answer]
@justb4 If you configure the `maxZoom` or `tileGrid` of your tile source correctly, this will work seamlessly. If you previously used `serverResolutions`, you'll want to create an `ol/TileGrid` with those as `resolutions` array, and use that as `tileGrid` for your tile source.

**Reply by justb4** [reply]
@ahocevar thanks for the very quick reply! Indeed, this works. For example, when the `maxZoom` set in the `View` of the `Map` is larger than the max index of the `TileGrid.resolutions` array.
